src/sidemenu/BoardMenuItem.tsx

```diff
@@ -31,21 +31,18 @@ const BoardMenuItem: React.FC<BoardMenuItemProps> = (props) => {
     current,
   } = props;
 
-  const notificationLabel =
-    props.muted
-    ? `${slug} muted`
-    :(props.updates && !props.outdated)
-    ? `${slug} has new updates`
-    :(props.updates && props.outdated)
-    ? `${slug} has updates`
-    : `${slug}`;
+  const getNotificationlabel = (props: BoardMenuItemProps) => {
+    if (props.muted) return `${props.slug} muted`;
```

Comment:
I'm torn about skipping the curly braces in the if conditions. In my memory, this is usually called out in style guides as a bad practice so I never do it, JS styleguides do allow it when it's a very short statement.

...let's keep it, but tif they're ever more complex than this, please err towards adding braces.

Reply:
Noted